addons/sale/static/src/js/sale_product_field.js

```diff
@@ -1,21 +1,19 @@
-/** @odoo-module **/
-
-import { _t } from "@web/core/l10n/translation";
+import {
+    ProductLabelSectionAndNoteField,
+    productLabelSectionAndNoteField,
+} from "@account/components/product_label_section_and_note_field/product_label_section_and_note_field";
```

Comment:
Are we sure we don't want to put framework stuff first and addons stuff after ? :thinking:

Reply:
AFAIK, there's no convention at odoo at the moment for javascript imports but we can set one in our team if you think it would be clearer

Reply:
Well, if you find a way to do so in a linter config it could be nice to have a logic similar to python imports, but if you cannot we can keep the standard alphabetical sort.